docs/install_guides/installing-openvino-windows.md

```diff
@@ -1,250 +1,150 @@
 
-3. **Optional:** You can choose to install Model Optimizer support for only certain frameworks. In the same directory are individual batch files for Caffe, TensorFlow, MXNet, Kaldi, and ONNX (`install_prerequisites_caffe.bat`, etc.). If you see error messages, make sure you installed all dependencies. 
-   
-The Model Optimizer is configured for one or more frameworks.
-
-You have now completed all required installation, configuration and build steps in this guide to use your CPU to work with your trained models. 
-
-If you want to use a GPU or VPU, or update your Windows* environment variables, read through the <a href="#optional-steps">Optional Steps</a> section:
-
-- <a href="#install-gpu">Steps for Intel® Processor Graphics (GPU)</a>
-- <a href="#hddl-myriad">Steps for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs</a>
-- <a href="#Update-Path">Add CMake* or Python* to your Windows* environment variables</a><br>
-
-Or proceed to the <a href="#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
-
-## <a name="optional-steps"></a>Step 5 (Optional): Configure Inference on non-CPU Devices:
-
-### <a name="install-gpu"></a>Optional: Steps for Intel® Processor Graphics (GPU)
-
-> **NOTE**: These steps are required only if you want to use an Intel® integrated GPU.
-
-This section will help you check if you require driver installation. Install indicated version or higher.
-
-If your applications offload computation to **Intel® Integrated Graphics**, you must have the Intel Graphics Driver for Windows installed on your hardware.
-[Download and install the recommended version](https://downloadcenter.intel.com/download/30079/Intel-Graphics-Windows-10-DCH-Drivers). 
-
-To check if you have this driver installed:
-
-1. Type **device manager** in your **Search Windows** box and press Enter. The **Device Manager** opens.
-
-2. Click the drop-down arrow to view the **Display adapters**. You can see the adapter that is installed in your computer:
-   ![](../img/DeviceManager.PNG)
-
-3. Right-click the adapter name and select **Properties**.
+> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. Check your system environment variables, and add Python if necessary.
 
-4. Click the **Driver** tab to see the driver version. 
-   ![](../img/DeviceDriverVersion.PNG)
+The environment variables are set. Next, you can download some additional tools.
 
-You are done updating your device driver and are ready to use your GPU. Proceed to the <a href="#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+## <a name="model-optimizer">Step 3 (Optional): Download additional components
```

Comment:
Can we add here information on how to download OpenCV using a script supplied by Windows installer?

@mshabunin to provide instructions.